### DISCUSSION TITLE
sql文件打开中文乱码

### DISCUSSION THREAD

**Discussion by exmono** [hhyo/Archery#2003] [Q&A]
![1F9A33E2-3FC8-41B1-A620-634F142B56C7](https://user-images.githubusercontent.com/53323105/208391602-466ac95d-c89f-44b8-9891-cf02776a63eb.png)

sql文件打开来，里面中文字符串是乱码，想问一下要做什么设置吗？请求帮忙，谢谢

**Comment by nick2wang** [comment]
sql文件需要使用utf8编码格式

**Reply by exmono** [reply]
谢谢，可以了
